app/coin/[id]/page.tsx

```diff
@@ -1,19 +1,170 @@
+                        )}
+                      </span>
+                    </div>
+                    <div className="text-sm dark:font-thin">
+                      {formatDate(currentData.market_data.ath_date[currency])}
+                    </div>
+                  </div>
+                  <div>
+                    <div className="flex flex-wrap items-center gap-3">
+                      <IoMdArrowDropdown className="text-4xl text-red-negative" />
+                      <h3 className="font-semibold dark:font-normal">
+                        All time low:
+                      </h3>{" "}
+                      <div className="text-xl font-bold">
+                        {formatCurrency(
+                          currentData.market_data.atl[currency],
+                          6,
+                          false,
+                          currency
+                        )}
+                      </div>
+                    </div>
+                    <div className="text-sm dark:font-thin">
+                      {formatDate(currentData.market_data.atl_date[currency])}
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div className="grid grid-cols-2 p-10 rounded-lg dark:bg-purple-market bg-purple-secondary">
+              <div className="flex flex-col items-center justify-center gap-7">
+                <h2 className="dark:font-thin">Market Cap</h2>
+                <p className="font-semibold">
+                  {formatCurrency(
+                    currentData.market_data.market_cap[currency],
+                    5,
+                    true,
+                    currency
+                  )}
+                </p>
+                <h2 className="dark:font-thin">Fully Diluted Valuation</h2>
+                <p className="font-semibold">
+                  {formatCurrency(
+                    currentData.market_data.fully_diluted_valuation[currency],
+                    5,
+                    true,
+                    currency
+                  )}
+                </p>
+                <h2 className="dark:font-thin">Volume/Market</h2>
+                <p className="font-semibold">
+                  {formatCurrency(
+                    currentData.market_data.market_cap[currency] /
+                      currentData.market_data.total_volume[currency],
+                    5,
+                    true,
+                    currency
+                  )}
+                </p>
+              </div>
+              <div className="flex flex-col items-center justify-center gap-7">
+                <h2 className="dark:font-thin">Total Volume</h2>
+                <p className="font-semibold">
+                  {formatCurrency(
+                    currentData.market_data.total_volume[currency],
+                    5,
+                    true,
+                    currency
+                  )}
+                </p>
```

Comment:
you could extrapolate this into a component cause you;re reusing it a few times. It can be done in the same file